src/pages/NewStory.js

```diff
@@ -148,7 +179,7 @@ const NewStory = () => {
                 Description: "${data.description}"
                 Title: "${data.title}"
                 Category: ${data.category}
-                user_story_status: "5f0f33205f5695666b0d2e7e"
+                user_story_status: "60b5cef600971013c4f269c2"
```

Comment:
what is this value? can we have a map of status somewhere rather than hardcoding these values please?